### DISCUSSION TITLE
Add SSL to a subdomain

### DISCUSSION THREAD

**Discussion by SkilldAI** [vercel/community#5292] [Help] [answered]
### Summary

Hello - we need to add a SSL  certificate to our subdomain - interviewhero.guru - the domain was purchased on namecheap
I have bought a SSL certificate from namecheap as well - but cant figure out how to add the ssl to the subdomain - chat.interviewhero.guru -

### Steps to Reproduce

how to add ssl?

**Comment by swarnava** [comment, marked as answer]
Hi,

Is chat.interviewhero.guru hosted in Vercel? Once you add the domain in Vercel dashboard project, it should automatically generate SSL certificate from Vercel. https://vercel.com/docs/projects/domains/working-with-domains#subdomain

**Reply by SkilldAI** [reply]
Hi - thanks for the quick revert - no its not hosted yet - i have read the document but couldnt figure out how to do it :-(

**Reply by swarnava** [reply]
Here is step by step article that may come handy: https://vercel.com/docs/projects/domains/add-a-domain#add-and-configure-domain

**Reply by SkilldAI** [reply]
<img width="613" alt="Screenshot 2023-12-26 at 10 51 48 AM" src="https://github.com/vercel/community/assets/145528336/682692b6-5ee6-41f0-bbe9-de5845993418">

i get this error when i try to add the A record for chat.interviewhero.guru - the ip is from our AWS server -

**Reply by swarnava** [reply]
Please make sure the IP address is an IPV4 address: https://support.dnsimple.com/articles/a-record/#record-format

**Comment by SkilldAI** [comment]
<img width="1295" alt="Screenshot 2023-12-27 at 1 43 03 PM" src="https://github.com/vercel/community/assets/145528336/58f59a74-a0fe-430d-8b57-29e6b06574fd">

 this is what we get now -

**Reply by swarnava** [reply]
Can you share the domain please?

**Reply by SkilldAI** [reply]
https://www.interviewhero.guru/
